src/sentry/api/endpoints/organization_events_spans_performance.py

```diff
@@ -295,10 +300,14 @@ def query_example_transactions(
             "id",
             "array_join(spans_op)",
             "array_join(spans_group)",
-            *(column.suspect_example_column for column in SPAN_PERFORMANCE_COLUMNS.values()),
+            *(
+                col
+                for column in SPAN_PERFORMANCE_COLUMNS.values()
+                for col in column.suspect_example_columns
```

Comment:
nit: can we avoid the double list comprehension, and just have the two for loops before the builder construction instead?